Akash/html/assign 2/assign2.html

```diff
@@ -5,10 +5,10 @@
             div{background-color: palevioletred;}
```

Comment:
it shouldbe like div  {
                             background-color: palevioletred;
                         }
make this change in each file.